projects/robots/k-team/khepera1/controllers/tcpip/tcpip.c

```diff
@@ -251,8 +251,8 @@ static void run() {
   } else if (buffer[0] == 'G') { /* set the position counter */
     int left, right;
     sscanf(buffer, "G,%d,%d", &left, &right);
-    left_encoder_offset = 50.0 * (wb_position_sensor_get_value(left_position_sensor) / M_PI) - left;
-    right_encoder_offset = 50.0 * (wb_position_sensor_get_value(right_position_sensor) / M_PI) - right;
+    // left_encoder_offset = 50.0 * (wb_position_sensor_get_value(left_position_sensor) / M_PI) - left;
+    // right_encoder_offset = 50.0 * (wb_position_sensor_get_value(right_position_sensor) / M_PI) - right;
```

Comment:
Does someone remember why we have these unused variables here?

Reply:
No, we should simply remove them.